### ISSUE TITLE
Wont open files

### ISSUE THREAD

**Issue by ghost** [scottanderson/railroad.studio#28]
Anytime I go to edit my file, I'll do as it says. Open the game directory and the save files. I click on the save file I want and get this message every time. 
![image](https://github.com/scottanderson/railroad.studio/assets/20000795/d6397eb8-7c94-486e-9254-13177712a83c)

**Comment by scottanderson** [comment]
Could you please share an affected save file?

**Comment by ghost** [comment]
Github doesnt let me upload the SAV File

**Comment by scottanderson** [comment]
You can attach a zip with the .sav file included.

**Comment by ghost** [comment]
[Save that wont load.zip](https://github.com/scottanderson/railroad.studio/files/12811756/Save.that.wont.load.zip)

**Comment by voidbeast** [comment]
It's the new map. I get the same error when loading a save from the lake valley map. The pine valley map saves still work as expected.

**Comment by voidbeast** [comment]
It's the new map. I get the same error when loading a save from the Lake Valley map. The pine valley map saves still work as expected. 

At first I posted this comment before thinking to check whether they had changed the format on all save files, so I went back and opened one of my saves from a Pine Valley map, then saved it again to make sure it was saved with the most recent version. And it still works on railroad.studio as expected. So pretty sure it's saves from the new map causing this.
